static/js/message_edit.js

```diff
@@ -165,6 +165,12 @@ exports.save = function (row, from_topic_edited_only) {
             request.propagate_mode = selected_topic_propagation;
         }
         changed = true;
+        var oldest_topic_message_visible_to_client = current_msg_list
+            .get_first_message_of_stream_topic(
+                message.stream,
+                message.topic
+            );
+        request.client_oldest_message_id = oldest_topic_message_visible_to_client.id;
```

Comment:
We generally avoid use this line-wrapping strategy since it's kinda confusing; I think just spelling this as:

```
var top_target_message = current_msg_list.get_first_message_of_stream_topic(
￼	                message.stream,
￼	                message.topic
￼	            );
￼	
```